group-generators/helpers/data-providers/otterspace/index.ts

```diff
@@ -0,0 +1,52 @@
+// eslint-disable-next-line no-restricted-imports
+import { gql } from "graphql-request";
+// eslint-disable-next-line no-restricted-imports
+import { SubgraphHostedServiceProvider } from "../subgraph";
+import { BadgeId, SubgraphRespnse } from "./types";
+import { FetchedData } from "topics/group";
+
+export class OtterspaceProvider {
```

Comment:
Maybe here you can inspire you from what that has already be done in the [POAP Provider](https://github.com/sismo-core/sismo-hub/blob/main/group-generators/helpers/data-providers/poap/base-provider.ts)
By extending your class to SubgraphHostedServiceProvider